Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -51,42 +53,49 @@ void InitClient()
             }
         }
 
-        // The client just uses the first transport available
-        Transport GetAvailableTransport()
+        public override void ClientConnect(Uri uri)
         {
             foreach (Transport transport in transports)
             {
-                if (transport.Available())
+                try
+                {
+                    if (transport.Available())
+                    {
+                        transport.ClientConnect(uri);
```

Comment:
Yeah,  I thought of using our own exception,  I think that will be better too.

If we don't want to use exceptions for this,  we could pass the schema to the "Available()"  method,  so that the transports can determine if they can handle the uri or not.